src/main/java/io/cdap/plugin/cdc/common/DBUtils.java

```diff
@@ -179,6 +208,38 @@ public static Object transformValue(int sqlType, int precision, int scale,
     return original;
   }
 
+  /**
+   * De-register all SQL drivers that are associated with the class
+   */
+  public static void deregisterAllDrivers(Class<? extends Driver> driverClass)
+    throws NoSuchFieldException, IllegalAccessException, ClassNotFoundException {
+    Field field = DriverManager.class.getDeclaredField("registeredDrivers");
+    field.setAccessible(true);
+    List<?> list = (List<?>) field.get(null);
+    for (Object driverInfo : list) {
+      Class<?> driverInfoClass = DBUtils.class.getClassLoader().loadClass("java.sql.DriverInfo");
+      Field driverField = driverInfoClass.getDeclaredField("driver");
+      driverField.setAccessible(true);
+      Driver d = (Driver) driverField.get(driverInfo);
+      if (d == null) {
+        LOG.debug("Found null driver object in drivers list. Ignoring.");
```

Comment:
Can this be trace? Also, the message can be "Could not find driver %s", driverInfo.toString()